src/SourceCompile/generate_parser_listener.tcl

```diff
@@ -17,7 +17,7 @@
 
 # Optional parameter to provide an output directory
 set DEST_DIR "."
-if [regexp {out_dir=([a-zA-Z0-9_/\.-]+)} $argv tmp DEST_DIR] {}
+if [regexp {out_dir=([a-zA-Z0-9_/\.-:]+)} $argv tmp DEST_DIR] {}
```

Comment:
Swap - and : 

if [regexp {out_dir=([a-zA-Z0-9_/\.:-]+)} $argv tmp DEST_DIR] {}

Reply:
Any reasoning as to why local builds on both Unix and Windows succeeded but ones on the test machines failed?

Reply:
Initially the build was failing and after my made my change it started working on Windows. So, I am certain I did build everything, not just once but multiple times. The version differences might be causing the error. On my machine its 8.6.8 and on the builder it's 8.6.0+9.